packages/api/core/test/fixture/app-with-maker-disable/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "test",
+  "config": {
+    "forge": {
+      "makers": [
+        {
+          "name": "@electron-forge/maker-zip",
+          "skip": true,
```

Comment:
Is it `skip` or `disabled`?

Reply:
[9ea965b](https://github.com/electron-userland/electron-forge/pull/2754/commits/9ea965b28f6eefd2bb13f52c3a883533ada5ae29) fixes this oopsies